pkg/capabilities/v2/protoc/pkg/pb/cre_metadata.proto

```diff
@@ -11,6 +11,7 @@ import "workflows/sdk/v2/pb/sdk.proto";
 message CapabilityMetadata {
   cre.sdk.v2.Mode mode = 1;
   string capability_id = 2;
+  bool supports_legacy_trigger_api = 3;
```

Comment:
I was thinking this could be another option that lives on the method called `leagacy_default` or something similar